jumanji/training/configs/env/tetris.yaml

```diff
@@ -0,0 +1,26 @@
+name: tetris
+registered_version: Tetris-v0
+
+network:
+    conv_num_channels: 32
+    tetromino_layers: 16
+    final_layer_dims: [64, 40]
```

Comment:
Could we make these configs environment-size-agnostic? As in, if I want 20 columns instead of 10, hardcoding 40 as the last `final_layer_dims` will break. Could we instead not pass in 40 and when instantiating the network, use the environment's `num_cols` attribute to infer what the size of the last layer should be?